Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Set the base image to Python
+FROM python:2.7.9
+MAINTAINER code040@intelworks.com
+
+# Environment
+ENV DEBIAN_FRONTEND noninteractive
+
+# Create the working dir and set the working directory
+WORKDIR /
+
+# Install dependencies
+COPY ./requirements.txt /requirements.txt
+RUN pip install -r requirements.txt \
+    && pip install supervisor \
+    && pip install gunicorn
+
+# Install OpenTaxii
+RUN mkdir /opentaxii
+COPY ./ /opentaxii/
+RUN cd  /opentaxii && python setup.py install && rm -rf /opentaxii
```

Comment:
does it make sense to create virtualenv here? I don't know much about how Docker works